au4/src/projectscene/qml/Audacity/ProjectScene/clipsview/TracksClipsView.qml

```diff
@@ -63,6 +63,7 @@ Rectangle {
         timeline.init()
         playCursorController.init()
         tracksViewState.init()
+        project.load()
```

Comment:
It's better to do the opposite here, call init, in which to call load.
init need call it once.
Load can be called several times for some reason.

Reply:
fixed